jade-repl.el

```diff
@@ -197,26 +196,34 @@ When ERROR is non-nil, use the error face."
 
 (defun jade-repl-emit-console-message (message)
   "Emit a console message.
-MESSAGE is a map ((text . ) (level . ) (type . ) (description . ) ...)."
+MESSAGE is a map (see https://chromedevtools.github.io/debugger-protocol-viewer/1-1/Console/#type-ConsoleMessage).
```

Comment:
Well, not really, since jade-repl should not be coupled with Chrome's debugger protocol :)